### PR TITLE
Automatic structuring feature

### PR DESCRIPTION
**Pull request by mixedrays**
This feature allows quickly add new sections into project, no more index.php manual edit required. 
Just create a new folders/subfolders into the `markup/` directory, and they will be added into project automatically.

### CODE REVIEW

**Comment by bjankord** [comment]
Looking through this now.

File: functions.php [hunk 1]
```diff
@@ -1,59 +1,105 @@
...
-        echo htmlspecialchars(file_get_contents('markup/'.$type.'/'.$file));
-        echo '</code></pre>';
-        echo '</div>';
-        echo '</div>';
-    endforeach;
+    }
+    echo '</div>';
+  }
+
+  function renterTitleFromPath($path, $parent) {
```
Comment (bjankord) [inline functions.php]: I'm guessing this was supposed to be "renderTitleFromPath" ?

**Comment by bjankord** [comment]
This looks good to me! I'll update the "renterTitleFromPath" function to "renderTitleFromPath" and then merge this into master.

**Comment by bjankord** [comment]
[Merged manually](https://github.com/bjankord/Style-Guide-Boilerplate/commit/6fbc031403836f87166f322d2ca80c9b8c4e2924). Thanks for the contribution @mixedrays! This is a nice feature addition.

File: functions.php (hunk 1 above)
Comment (mixedrays) [inline functions.php]: Sure, i'ts misprint, my bad

Comment (bjankord) [inline functions.php]: No worries :)

**Pull request closed without merging**